util/src/test/java/com/google/cloud/hadoop/util/HttpTransportFactoryTest.java

```diff
@@ -15,17 +15,33 @@
 package com.google.cloud.hadoop.util;
 
 import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertSame;
 import static org.junit.Assert.assertThrows;
+import static org.mockito.Mockito.when;
```

Comment:
Please use fakes and stubs - we avoid using mocks in GCS connector tests to avoid maintenance cost.